go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -740,6 +740,60 @@ var _ = Describe("User Defined Network Controller", func() {
 					Expect(kerrors.IsNotFound(err)).To(Equal(true))
 				}
 			})
+
+			DescribeTable("should report error in status, when namespace-selector",
+				func(selector metav1.LabelSelector, expectedMessage string) {
+					cudn := &udnv1.ClusterUserDefinedNetwork{
+						ObjectMeta: metav1.ObjectMeta{Name: "test-cudn"},
+						Spec:       udnv1.ClusterUserDefinedNetworkSpec{NamespaceSelector: selector},
+					}
+					c = newTestController(template.RenderNetAttachDefManifest, cudn)
+					Expect(c.Run()).To(Succeed())
```

Comment:
nit: couldn't you inline these ? Are you actually using `c` anywhere else ? Just curious, you can leave it as is.

Reply:
Unfortunately we cannot inline it because there is some AfterEach code that shutdown the tested controller instance.
Without this cleanup unit-tests would fail due to race errors (e.g.: multiple places touches OVN config).

Reply:
Right, thanks for explaining; I didn't check the entire code.